src/components/DayPickerNavigation.jsx

```diff
@@ -121,6 +129,7 @@ function DayPickerNavigation({
           {...css(
```

Comment:
@backwardok how do we mark a fake button as disabled? I'd like to do that in this scenario so that there's no confusion. `aria-disabled` maybe?

Reply:
Yep, `aria-disabled` would be the right prop (https://www.w3.org/TR/wai-aria/#aria-disabled)